src/System.Device.Ports/System.Device.Ports.SerialPort/xlf/Strings.de.xlf

```diff
@@ -2,11 +2,36 @@
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="de" original="../Strings.resx">
     <body>
+      <trans-unit id="Arg_InvalidSerialPort">
+        <source>The given port name ({0}) does not resolve to a valid serial port</source>
+        <target state="new">The given port name ({0}) does not resolve to a valid serial port</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Arg_InvalidSerialPortExtended">
+        <source>The given port name is invalid.  It may be a valid port, but not a serial port</source>
+        <target state="new">The given port name is invalid.  It may be a valid port, but not a serial port</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Arg_WrongAsyncResult">
+        <source>IAsyncResult object did not come from the corresponding async method on this type</source>
+        <target state="new">IAsyncResult object did not come from the corresponding async method on this type</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="ArgumentOutOfRange_Bounds_Lower_Upper">
+        <source>Argument must be between {0} and {1}</source>
+        <target state="new">Argument must be between {0} and {1}</target>
+        <note />
+      </trans-unit>
       <trans-unit id="ArgumentOutOfRange_Enum">
         <source>Enum value was out of legal range</source>
         <target state="new">Enum value was out of legal range</target>
         <note />
       </trans-unit>
```

Comment:
Did you incorrectly copy the sources of these files? There are no translations in here, neither in the new strings (which would be expected) nor in the existing ones. 

Maybe cross-check at the end. If we don't have any translations, we shouldn't generate the satellite assemblies either.